browser/help/topics/v4-hts.md

```diff
@@ -25,52 +25,614 @@ The '`freq`' annotation is an array, and each element of the array is a struct t
-3. For the HGDP and 1KG subsets in the gnomAD v4.0 genomes,, project specified populations are available in place of gnomAD inferred global populations. The HGDP populations are detailed here. The 1KG populations are described here
+
+Global fields
+
+- `versions`: Struct containing the exomes and genomes versions.
+- `genomes_globals`: Global fields from the gnomAD genomes.
+  - `freq_meta`: Allele frequency metadata for the gnomAD genomes. An ordered list containing the frequency aggregation group for each element of the `genomes.freq` array row annotation.
+  - `freq_meta_sample_count`: A sample count per sample grouping defined in the genomes `genomes.freq_meta` global annotation.
+  - `faf_meta`: Filtering allele frequency metadata for the gnomAD genomes. An ordered list containing the frequency aggregation group for each element of the `genomes.faf` array row annotation.
+  - `age_distribution`: Callset-wide age histogram calculated on the gnomAD genomes.
+    - `bin_edges`: Bin edges for the age histogram.
+    - `bin_freq`: Bin frequencies for the age histogram. This is the number of records found in each bin.
+    - `n_smaller`: Count of age values falling below lowest histogram bin edge.
+    - `n_larger`: Count of age values falling above highest histogram bin edge.
+- `exomes_globals`: Global fields from the gnomAD exomes.
+  - `freq_meta`: Allele frequency metadata for the gnomAD exomes. An ordered list containing the frequency aggregation group for each element of the `exomes.freq` array row annotation.
+  - `freq_meta_sample_count`: A sample count per sample grouping defined in the exomes `exomes.freq_meta` global annotation.
+  - `faf_meta`: Filtering allele frequency metadata for the gnomAD exomes. An ordered list containing the frequency aggregation group for each element of the `exomes.faf` array row annotation.
+  - `age_distribution`: Callset-wide age histogram calculated on the gnomAD exomes.
+    - `bin_edges`: Bin edges for the age histogram.
+    - `bin_freq`: Bin frequencies for the age histogram. This is the number of records found in each bin.
+    - `n_smaller`: Count of age values falling below lowest histogram bin edge.
+    - `n_larger`: Count of age values falling above highest histogram bin edge.
+- `joint_globals`: Global fields from the combined (joint) gnomAD exomes and genomes.
+  - `freq_meta`: Allele frequency metadata for the joint gnomAD exomes and genomes. An ordered list containing the frequency aggregation group for each element of the `joint.freq` array row annotation.
+  - `freq_index_dict`: Dictionary keyed by specified label grouping combinations (group: adj/raw, gen_anc: gnomAD inferred genetic ancestry group, sex: sex karyotype), with values describing the corresponding index of each grouping entry in the `joint.freq` array row annotation.
```

Comment:
@jkgoodrich `exomes_globals` and `genomes_globals` don't have `freq_index_dict` nor `faf_index_dict`, do we need to add them in?

Reply:
resolving this now that you've added them